sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpExclusionsFileFilter.java

```diff
@@ -19,19 +19,28 @@
  */
 package org.sonar.plugins.php;
 
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
 import org.sonar.api.batch.fs.InputFile;
 import org.sonar.api.batch.fs.InputFileFilter;
 import org.sonar.api.config.Configuration;
 import org.sonar.api.utils.WildcardPattern;
 import org.sonar.api.utils.log.Logger;
 import org.sonar.api.utils.log.Loggers;
+import org.sonar.php.parser.LexicalConstant;
 import org.sonar.plugins.php.api.Php;
 
 public class PhpExclusionsFileFilter implements InputFileFilter {
 
   private final String[] excludedPatterns;
-
   private static final Logger LOG = Loggers.get(PhpExclusionsFileFilter.class);
+  private static final int DEFAULT_AVERAGE_LINE_LENGTH_THRESHOLD = 220;
```

Comment:
why not `200` as specified in the ticket?

Reply:
nicer with ruling on symfony (one less file excluded)